app/models/student.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+  attr_accessible :title, :first_name, :middle_name, :last_name, :email, :birth_date, :gender end
+
+  # allow for null values where applicable to allow for single names
+  # allow for null values where applicable since you cannot legally force a user to identify their gender
+  validates :title, length: {maximum => 10 }
+  validates :last_name, :length: { maximum => 50 } , presence: { message: "Please enter your family name" }
+  validates :first_name, :middle_name, :length: { maximum => 50 }
+  validates :email, presence: { message: "Please enter your email address"  }, length: {maximum => 100}
+
+  # allow for 'not stated', male, female, and other
+  validates :gender :inclusion => { :in => %w(@ m f x) }
```

Comment:
I think there's a small syntax issue here where the comma is skipped between `:gender` and `:inclusion`. I would also make this use the "newer ruby object syntax". Also, I think using `@` for "not stated" might be an avetmiss specific thing? (or maybe education specific thing?), rather than use a magic string we could leave it as `nil` (or `NULL`) if the user prefers not to say

```ruby
validates :gender, inclusion: { in: %w(m f x) }, allow_nil: true
```

and we could always have a function later that renders the `@` for us


```ruby
# in app/models/user.rb
def format_gender
  if gender.present? 
    gender
  else
   "@"
  end
end